### PR TITLE
Mapped most of the new 20w16 stuff

### PR DESCRIPTION
**Pull request by Shnupbups**
Mapped most of the new 20w16 stuff, barring the JMX things

### CODE REVIEW

File: mappings/net/minecraft/structure/BastionTreasureData.mapping [hunk 1]
```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/class_5182 net/minecraft/structure/BastionTreasureData
```
Comment (YanisBft) [inline mappings/net/minecraft/structure/BastionTreasureData.mapping:1]: `TreasureBastionData` or even `TreasureBastionRemnantData`, to follow how we named village data (`DesertVillageData`, etc)

File: mappings/net/minecraft/structure/BastionGenerator.mapping [hunk 2]
```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/class_5179 net/minecraft/structure/BastionGenerator
```
Comment (YanisBft) [inline mappings/net/minecraft/structure/BastionGenerator.mapping:1]: The structure is called "bastion remnant", so `BastionRemnantGenerator`

File: mappings/net/minecraft/structure/BastionTreasureData.mapping (hunk 1 above)
Comment (Runemoro) [reply mappings/net/minecraft/structure/BastionTreasureData.mapping:1]: "Desert village" is a type of village, but "treasure bastion" isn't. All bastions have treasure, and this class stores the data for the treasure of the bastion.

File: mappings/net/minecraft/structure/HoglinStableData.mapping [hunk 3]
```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/class_5177 net/minecraft/structure/HoglinStableData
```
Comment (Runemoro) [inline mappings/net/minecraft/structure/HoglinStableData.mapping:1]: What does "stable" mean here? I also think it should be `StableHoglinData`.

Comment (liach) [reply mappings/net/minecraft/structure/HoglinStableData.mapping:1]: stable is like the place where you raise a horse. this is like a hoglin farm.

Comment (YanisBft) [reply mappings/net/minecraft/structure/HoglinStableData.mapping:1]: It's stable as a place. The class stores the data of the hoglin stable building

**Comment by liach** [comment]
Mind fix merge conflicts @Shnupbups?

File: mappings/net/minecraft/network/packet/c2s/play/GenerateJigsawC2SPacket.mapping [hunk 4]
```diff
@@ -0,0 +1,8 @@
+CLASS net/minecraft/class_5194 net/minecraft/network/packet/c2s/play/GenerateJigsawC2SPacket
```
Comment (liach) [inline mappings/net/minecraft/network/packet/c2s/play/GenerateJigsawC2SPacket.mapping:1]: Mind remove this class? this is already named to `JigsawGeneratingC2SPacket`

File: mappings/net/minecraft/structure/BastionGenerator.mapping (hunk 2 above)
Comment (liach) [reply mappings/net/minecraft/structure/BastionGenerator.mapping:1]: Yes `BastionRemnantGenerator` rename please

**Review by liach — CHANGES_REQUESTED** [review]

**Comment by liach** [comment]
Superseded by #1288.

**Pull request closed without merging**